bin/geo2svg

```diff
@@ -42,6 +44,7 @@ writer.write("<?xml version=\"1.0\" encoding=\"utf-8\"?>" + os.EOL
     + " viewBox=\"0 0 " + +commander.width + " " + +commander.height + "\""
     + (commander.fill != "black" ? " fill=\"" + attr(commander.fill) + "\"" : "")
     + (commander.stroke != "none" ? " stroke=\"" + attr(commander.stroke) + "\"" : "")
+    + (commander.fontFamily != "none" ? " font-family=\"" + attr(commander.fontFamily) + "\"" : "")
```

Comment:
How about adding a configurable default font-size, too?

And probably a configurable default text-anchor.